xml/cioos_template.xml

```diff
@@ -1,49 +1,125 @@
+{% macro multi_lang(record_key) %}
+{% set alternates = get_alternate_text(record_key) %}
+{% if record[record_key] %}
+  {# text in the default language #}
+  <gco:CharacterString>{{ record[record_key] }}</gco:CharacterString>
+{% endif %}
+{% if alternates | length > 0 %}
+  <lan:PT_FreeText>
+    {% for lang_code,lang_text in alternates %}
```

Comment:
Does this ensure the default language and alternate language is always the same throughout the file? It shouldn't be possible to use _eng and _fra alternates in the same input record.yaml. Might need a separate test for this?